Bylaws.md

```diff
@@ -48,34 +52,37 @@ Chartered Working Group (CWG) Members are automatically granted Forum Membership
    * Certificate Issuer: The member organization operates a certification authority that meets the requirements for membership as a Certificate Issuer in one or more CWGs.
-   * Root Certificate Issuer: The member organization operates a certification authority that meets the requirements for membership as a Root Certificate Issuer in one or more CWGs.
-   * Certificate Consumer: The member organization meets the requirements for membership as a Certificate Consumer in one or more CWGs.
-* Associate Member: As defined in Section 3.1
-* Interested Party: As defined in Section 3.2
+   * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.
+* Other types of membership consists of the following three (3) sub-categories:
+   * Associate Member: As defined in Section 3.1
+   * Interested Party: As defined in Section 3.2
+   * Probationary Member (Certificate Issuer/Certificate Consumer): As defined in Section 3.3.
+
+Note: Based on Section 3.1 in version 2.4 of the Bylaws, it was allowed for a Certificate Issuer that partially met the CWG membership criteria to be accepted as "Associate Member". Once version 2.5 of the Bylaws becomes effective, these Associate Members will automatically be re-assigned to the "Probationary Member" membership category, as described in Section 3.3 below.
 
-Voting Members that qualify under more than one category of Forum Membership MAY choose their membership category.
+Voting Members that qualify under more than one category of Forum membership must choose only one membership category at the Forum level. Similarly, a Voting Member within a given CWG that qualifies under more than one category of CWG membership must choose only one membership category in that CWG.
 
 ## 2.2 Ending Forum Membership
 
-When a Forum Member ceases to be a Member of at least one CWG, their Forum Membership shall be terminated. Termination does not prevent a Member from potentially having continuing obligations, under the Forum's IPR Policy or any other document.
+When a Forum Member ceases to be a Member of at least one CWG, its Forum membership is terminated. Termination does not prevent a Member from potentially having continuing obligations, under the Forum's IPR Policy or any other document.
 
 ## 2.3 General Provisions Applicable to All Ballots
 
 The following rules will apply to all ballots, including Draft Guideline Ballots (defined in 2.4).
 
  1. Only votes by Voting Members shall be accepted.
   
- 2. Only one vote per Voting Member company shall be accepted; Affiliates of Voting Members shall not vote. Only votes from Voting Representatives shall be accepted.
+ 2. Only one vote per Voting Member company shall be accepted; Affiliates of Voting Members shall not vote. Only votes from a designated representative of a Voting Member ("Voting Representative") shall be accepted.
```

Comment:
Nit: Voting Members are referenced as “organizations” in Section 2.1. Suggest changing “company” to “organization” for consistency.

Reply:
Fixed.